lib/rules/rule-selector-property-disallowed-list/index.mjs

```diff
@@ -49,14 +49,18 @@ const rule = (primary) => {
 				return;
 			}
 
-			ruleNode.walkDecls((decl) => {
-				const { prop } = decl;
+			ruleNode.walk((node) => {
+				if (node.type !== 'decl') return;
```

Comment:
[suggest] Is this the same as the original code?

```suggestion
			ruleNode.walkDecls((decl) => {
```

Reply:
Ah, yes, I changed it back and forth a couple of times, but the original is better and is what we want.